browser/main/index.js

```diff
@@ -73,6 +73,9 @@ ReactDOM.render((
           <IndexRedirect to='/alltags' />
           <Route path=':tagname' />
         </Route>
+        <Route path='narrowToTag'>
```

Comment:
What is this for?

Reply:
Just forgot to remove, thanks for pointing it out!